controllers/ProductController.php

```diff
@@ -31,7 +31,7 @@ public function actionIndex()
             ]);
         } else {
             $products = Product::find()->where([
-                'created_by' => Yii::$app->user->id,
+                'created_by' => $this->contentContainer->id,
```

Comment:
`$this->contentContainer` this is the space object right? if so that page will show all the products of that space, we don't want that, we want to show all the products of the user that you're visiting, or at least that the logic of it if I'm not mistaken..

Reply:
No , `$this->contentContainer` can be the user object or the space object , have a deep look here : 
http://docs.humhub.org/dev-content.html